### PR TITLE
itR-8: feat: add footer

### PR DESCRIPTION
**Pull request by MilaKolasava**
image
image

### CODE REVIEW

File: src/components/Jumbotron/Jumbotron.css [hunk 1]
```diff
@@ -1,8 +1,11 @@
-.main-picture {
+.main {
   background: url(../../assets/Open-Air-Gampel.jpg), center, no-repeat;
+}
+
+.big-picture {
   margin: 0 auto;
   width: 100vw;
-  height: calc(100vh - 31px);
+  height: calc(100vh - 31px - 34px);
```
Comment (glebpigulevsky) [inline src/components/Jumbotron/Jumbotron.css]: simplify it please, it's pure math.

File: src/components/Jumbotron/Jumbotron.css [hunk 2]
```diff
@@ -1,8 +1,11 @@
-.main-picture {
+.main {
   background: url(../../assets/Open-Air-Gampel.jpg), center, no-repeat;
+}
+
+.big-picture {
```
Comment (glebpigulevsky) [inline src/components/Jumbotron/Jumbotron.css]: if you consider it as sharable styles you have to move it in the common file with styles

File: src/components/Footer/Footer.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import "./Footer.css";
+import { ReactComponent as LinkedIn } from "../../assets/linkedin-icon.svg";
+import { ReactComponent as GitHub } from "../../assets/github-icon.svg";
+
+function Footer() {
+  return (
+    <nav className="wrapper" id="footer" data-testid="footer-wrapper">
+      <a href="http://www.linkedin.com/in/luydmila-kolasava-1940a01b0">
```
Comment (glebpigulevsky) [inline src/components/Footer/Footer.tsx]: please add ability to open links in the new tab

**Review by glebpigulevsky — CHANGES_REQUESTED** [review]
good job. but requires little bit clean up

**Comment by glebpigulevsky** [comment]
could you provide screenshot please of work you have done?

File: src/components/Jumbotron/Jumbotron.css (hunk 1 above)
Comment (MilaKolasava) [reply src/components/Jumbotron/Jumbotron.css]: done

File: src/components/Footer/Footer.tsx (hunk 3 above)
Comment (MilaKolasava) [reply src/components/Footer/Footer.tsx]: done

File: src/components/Jumbotron/Jumbotron.css (hunk 2 above)
Comment (MilaKolasava) [reply src/components/Jumbotron/Jumbotron.css]: done

**Review by glebpigulevsky — APPROVED** [review]

**Pull request merged**